### ISSUE TITLE
Literate Haskell not supported

### ISSUE THREAD

**Issue by fmease** [boyter/scc#204]
[Literate Haskell](https://wiki.haskell.org/Literate_programming) is an alternative form of Haskell code where comments are the default and code marked special. It's not uncommon.
The file extension is `.lhs`. The data of Haskell and Literate Haskell should be merged in the final output.
For prior art, see cloc.

**Comment by boyter** [comment]
Yeah this is pretty close to https://github.com/boyter/scc/issues/43 as an issue, since it inverts the way its normally counted. Although the merging into Haskell is another consideration.

Ill have a think on this one because its not a trivial case to add.
